test/test_xnnpack_integration.py

```diff
@@ -479,24 +479,52 @@ def __init__(self):
                 self.paddings = paddings
                 self.dilations = dilations
                 self.groups = groups
+                self.activation_fn = F.relu
+                if (inplace):
+                    self.activation_fn = F.relu_
 
             def forward(self, x):
                 o = F.conv2d(x, self.conv_weight, self.conv_bias,
                              self.strides, self.paddings, self.dilations, self.groups)
+                o = self.activation_fn(o)
                 o = o.permute([0, 2, 3, 1])
                 o = F.linear(o, self.linear_weight, self.linear_bias)
-                return F.relu(o)
+                return self.activation_fn(o)
 
         pattern_count_map = {"Tensor = aten::conv2d": -1,
                              "prepacked::conv2d_clamp_prepack": 1,
                              "prepacked::conv2d_clamp_run": 1,
-                             "Tensor = prim::CallFunction": -1,
                              "prepacked::linear_clamp_prepack": 1,
                              "prepacked::linear_clamp_run": 1}
-        validate_transformed_module(M, pattern_count_map, data_shape)
+        validate_transformed_module(M(), pattern_count_map, data_shape)
+        pattern_count_map["prepacked::conv2d_clamp_prepack"] = -1
+        pattern_count_map["Tensor = prim::CallFunction"] = -1
+        pattern_count_map["prepacked::linear_clamp_prepack"] = -1
+        validate_transformed_module(M(), pattern_count_map, data_shape, True)
+
+        # Not inplace relu fusion test.
+        pattern_count_map = {"aten::relu": 2,
+                             "prepacked::conv2d_clamp_prepack": -1,
+                             "prepacked::conv2d_clamp_run": 1,
+                             "prepacked::linear_clamp_prepack": -1,
+                             "prepacked::linear_clamp_run": 1}
+        validate_transformed_module(M(), pattern_count_map, data_shape, True)
+        pattern_count_map["prepacked::conv2d_clamp_prepack"] = -1
+        pattern_count_map["prepacked::linear_clamp_prepack"] = -1
+        pattern_count_map["aten::relu"] = -1
+        validate_transformed_module(M(), pattern_count_map, data_shape, True, True)
```

Comment:
See https://our.intern.facebook.com/intern/wiki/Facebook-code-smells/boolean-arguments/ .  I think using booleans is fine here, but pass the args by keyword instead of position so it's clear what they mean.